### DISCUSSION TITLE
Displaying Adobe font error

### DISCUSSION THREAD

**Discussion by jiseon-baek** [wojtekmaj/react-pdf#1246] [Q&A]
Hello, I'm trying to display pdf with adobe myungjo std font.

I can't see this font when I render pdf. Is there anyone how to solve this font error for rendering?

**Origin**

![스크린샷 2023-01-02 오후 4 35 24](https://user-images.githubusercontent.com/83223501/210204398-d1e2637a-84ae-43b3-a518-66149bd18f46.png)

**After rendering in react-pdf** (I can't see any texts)
<img width="663" alt="스크린샷 2023-01-02 오후 4 37 21" src="https://user-images.githubusercontent.com/83223501/210204525-6b453b59-bf7c-4618-b941-c8c97c4ae3ed.png">

**Comment by wojtekmaj** [comment]
Can you share the PDF so I could have a look?
